x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -195,8 +196,7 @@ describe('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
         .and('be.enabled');
       cy.getByTestSubj(FLEET_REASSIGN_POLICY_MODAL_CONFIRM_BUTTON).click();
 
-      // ideally we should use applied policy instead: https://github.com/elastic/security-team/issues/8837
-      const policyIdPath = 'response.body.data[0].policy_info.agent.configured.id';
+      const policyIdPath = 'response.body.data[0].policy_info.agent.applied.id';
```

Comment:
Question: There are differences between the `applied` and the `configured` which I don't remember now - but if I remember correctly, they could differ. Can we make sure we understand the difference between the two from an Endpoint standpoint to ensure we are doing the correct check here

Reply:
`configured` is what the policy is configured in kibana to be whereas the `applied` is what is actually on the agent. the contents are ideally the same but `applied` will lag behind `configured` a bit. `applied` is preferred for the test since it is a more complete test. we were using `configured` here due to the policy not getting applied correctly occasionally in serverless tests which has now been resolved.

Reply:
Got it. Thanks for that. I think `applied` is also what we use in the UI to show the "out of date" tooltip icon on the policy version in the Endpoint's list